src/index.html

```diff
@@ -1,13 +1,435 @@
+          call to order
+        </a>
+      </div>
+    </aside>
+
+    <main class="main">
+      <section class="museum-hours" id="museum-hours">
+        <div class="museum-hours-container">
+          <p class="museum-hours__visit">Plan your visit</p>
+          <h2 class="section-title section-title--museum-hours">
+            Museum hours
+          </h2>
+        </div>
+
+        <div class="museum-hours__fifth-avenue">
+          <h3 class="section-title-second">The Met Fifth Avenue</h3>
+          <div class="museum-hours__info">
+            <div class="museum-hours__hours museum-hours--text-bold">
+              <p>10 am - 5 pm</p>
+              <p>10 am - 9 pm</p>
+              <p>Closed</p>
+            </div>
+
+            <div class="museum-hours__days">
+              <p>Sun, Mon, Tue, Thu</p>
+              <p>Fri, Sat</p>
+              <p>Wen</p>
+            </div>
+          </div>
+
+          <div class="museum-hours__closed-on">
+            <p class="museum-hours__closed-on-text museum-hours--text-bold">
+              Closed on
+            </p>
+
+            <p class="museum-hours__closed-on-info">
+              Thanksgiving Day, Dec 25, Jan 1, and the first Monday in May.
+            </p>
+          </div>
+        </div>
+
+        <div class="museum-hours__cloisters">
+          <h3 class="section-title-second section-title-second--cloisters">The Met Cloisters</h3>
+          <div class="museum-hours__info">
+            <div class="museum-hours__hours museum-hours--text-bold">
+              <p>10 am - 4.30 pm</p>
+              <p>Closed</p>
+            </div>
+
+            <div class="museum-hours__days">
+              <p>Thu - Tue</p>
+              <p>Wen</p>
+            </div>
+          </div>
+
+          <div class="museum-hours__closed-on">
+            <p class="museum-hours__closed-on-text museum-hours--text-bold">
+              Closed on
+            </p>
+
+            <p class="museum-hours__closed-on-info">
+              Thanksgiving Day, Dec 25, Jan 1
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="now-on-view" id="now-on-view">
+        <div class="now-on-view-container">
+          <h2 class="section-title section-title--now-on-view">Now On View</h2>
+          <p class="now-on-view--text-normal now-on-view--text-normal-centered">
+            Explore exhibition gallery guides, videos, articles, and more.
+          </p>
+        </div>
+
+        <article class="now-on-view-info">
+          <div class="now-on-view-photo-container">
+            <a href="#">
+              <div class="
+                  now-on-view__photo
+                  now-on-view__photo--background
+                  now-on-view__photo--background--first"
+              >
+              </div>
+            </a>
+
+            <a href="#">
+              <img
+                src="./images/Layer 47@1,5x.png"
```

Comment:
You have problems with photo, may be it becouse of ./ before images, try to delete this part